lua/lv-which-key/init.lua

```diff
@@ -198,27 +178,26 @@ if O.extras then
         t = {"<cmd>VimtexTocToggle<cr>", "Toggle Table Of Content"},
         v = {"<cmd>VimtexView<cr>", "View PDF"}
     }
+    normalMappings["p"] = {"<cmd>lua require'telescope'.extensions.project.project{}<cr>", "Projects"}
```

Comment:
Telescope is not an extra dependency, please group with the ones above.

Reply:
see comment below telescope-project is extra